src/clone.c

```diff
@@ -140,8 +222,13 @@ SYS_FUNC(clone)
 			printaddr(tcp->u_arg[ARG_CTID]);
 			tprint_arg_name_end();
 		}
+		if ((flags & nsflags) && !syserror(tcp)) {
+			tcp->auxstr = get_namespace_auxstr((int)tcp->u_rval, flags, tcp);
```

Comment:
We don't cast `tcp->u_rval` to `int` in another `get_namespace_auxstr` invocation, let's be consistent.
If there has to be a cast, it has to be in both places.  If not, then also in both places.

Reply:
fixed locally.